install.sh

```diff
@@ -2,41 +2,49 @@
 cd `dirname $0`
 set -e
 
+echo -n 'Looking for javac... '
 if [ ! -f /usr/bin/javac ] ; then
-    # Need to update or else the installs won't work
+    echo -n '... not found. Installing packages... '
+    # Need to update or else the installs won't work. Failing is not critical.
     sudo DEBIAN_FRONTEND=noninteractive apt-get -qq update
 
     # Downgrade tzdata, hilarious.
-    sudo DEBIAN_FRONTEND=noninteractive apt-get -qq install --force-yes -y tzdata/trusty
+    sudo DEBIAN_FRONTEND=noninteractive apt-get -qq install --force-yes -y tzdata/trusty || (echo Failed to install tzdata. ; exit 1)
```

Comment:
In theory I humbly request single quotes around these strings, too, but it's not going to block me merging it.